install/debian/preinst

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh -e
+
+NAME=fah-client
+
+case "$1" in
+  install)
+    # old v7 init script is broken and does not stop when told so
+    if [ -e /etc/init.d/FAHClient ]; then
+      systemctl -q kill FAHClient || true
```

Comment:
`kill` verb is silent by default. `-q` not needed.

Reply:
Done.